addons/observability/thanos.yaml

```diff
@@ -0,0 +1,7 @@
+type: s3
+config:
+  bucket: <BUCKET_NAME>
+  endpoint: s3.<CLOUD_REGION>.amazonaws.com
+  insecure: true
```

Comment:
@tpouyer can we change this to false please. ```insecure: false```. This will make the traffic between Hub and S3 https